assets/aml-benchmark/components/src/aml_benchmark/inference_postprocessor/inference_postprocessor.py

```diff
@@ -175,29 +175,24 @@ def read_ground_truth_dataset(self) -> pd.DataFrame:
             actual_df = pd.json_normalize(
                 read_jsonl_files(resolve_io_path(self.ground_truth_dataset))
             )
+            if not self.ground_truth_dataset and not self.additional_columns:
+                return actual_df
             if self.ground_truth_column_name:
                 result_df[self.ground_truth_column_name] = actual_df[
                     self.ground_truth_column_name
                 ]
             if self.additional_columns:
-                elements = self.additional_columns.split(",")
-                strips = [s.strip() for s in elements if s.strip()]
-                for column in strips:
-                    try:
-                        result_df[column] = actual_df[column]
-                    except KeyError:
-                        raise BenchmarkUserException._with_error(
+                columns = [col.strip() for col in self.additional_columns.split(',')]
```

Comment:
We also need to remove string that have become empty after stripping (like it was done before).

Reply:
Will raise a warning letting user know that they have provided empty string in the additional columns and then drop those empty strings from additional columns.